roboteam_ai/src/control/positionControllers/ForcePosControl.cpp

```diff
@@ -47,11 +48,13 @@ Vector2 ForcePosControl::calculateForces(const RobotPtr &robot, const Vector2 &t
             force += ControlUtils::calculateForce(Vector2(-1.0, -1.0) / robot->pos, FORCE_WEIGHT_FIELD_SIDES, 99.9);
         }
     }
-
+    force = (force.length() > 3.0) ? force.stretchToLength(3.0) : force;
```

Comment:
Yeah this is definitely not just a refactor. Do not like this.

Reply:
see above